kuma/banners/templatetags/jinja_helpers.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django_jinja import library
+
+from ..models import Banner
+
+
+@library.global_function
+def get_banners():
+    """
+    Get all active banners ordered by priority
+    """
+    return list(Banner.objects.filter(active=True).order_by('priority'))
```

Comment:
Actually, if `{% include "banners/banners.html" %}` is only ever called from `landing/homepage.html` then there *is* a great place to generate this context *before* rendering. 

And that's [`landing.view.home`](https://github.com/mozilla/kuma/blob/391774e15431971ea82aeac4e006f104a16f4d51/kuma/landing/views.py#L25)
Or, perhaps the `render_home` view function to benefit the React based one. 

Actually, why is the banner not displayed from `landing/react_homepage.html` also?

Reply:
Also, we'd need to conditional that on `waffle.flag('developer_needs')` in the view function and once that's in place there's no need to use `waffle.flag('developer_needs')` in the jinja template. You can just leverage the fact that `banners` is truthy (if waffled on and has a list of banners).

Reply:
> Actually, if `{% include "banners/banners.html" %}` is only ever called from `landing/homepage.html` then there _is_ a great place to generate this context _before_ rendering.

It is going to be added to other pages as well once the solution is more fully baked.

> And that's [`landing.view.home`](https://github.com/mozilla/kuma/blob/391774e15431971ea82aeac4e006f104a16f4d51/kuma/landing/views.py#L25)
> Or, perhaps the `render_home` view function to benefit the React based one.
> 
> Actually, why is the banner not displayed from `landing/react_homepage.html` also?

It is not yet essential to be added there, and there will be some additional JS rewriting required once it is moved to the React portion of the codebase. At the moment neither the React bits nor this banner is shown to users.

Reply:
So, it's going to be used on all pages? If so, it should move to a context processor. Happy to help if in doubt. 

If you just want to get this landed ASAP, then there should be a code comment pointing to the bug about rewriting it later.